<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modify_courses</class>
 <widget class="QDialog" name="modify_courses">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1431</width>
    <height>824</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="Text_courseName">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>60</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>170</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Instructors Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>390</y>
     <width>341</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>* to assign TA's click select the TA ID row that identifies the Ta
 *for multiple TA's click TA ID rows while holding the key Ctrl</string>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>390</y>
     <width>571</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_currentInfo">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>750</y>
     <width>151</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Load current information</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>60</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Couse Name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_load">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>700</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>load TAs</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Text_instructorEmployeeID">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>210</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>140</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>End Date</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Add_modify_studentList">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>360</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>load Student List</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="date_startDate">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>100</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Text_instructorName">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>170</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="date_endDate">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>140</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>733</x>
     <y>10</y>
     <width>20</width>
     <height>791</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QTableWidget" name="Ta_list">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>300</y>
     <width>211</width>
     <height>341</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <column>
    <property name="text">
     <string>TA NAME</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>TA ID</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="Courses">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>271</width>
     <height>721</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Course ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Couse Name</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="Student_List">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>60</y>
     <width>411</width>
     <height>251</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>First Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Middle Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Last Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Student ID</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="pushButton_A_M_studentList">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>360</y>
     <width>221</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add/Modify Student List</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Add_tas">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>760</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Add TA</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_LoadCourse">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>750</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Load courses</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>200</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Instrucor
EmployeerID</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Modify">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>750</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>modify </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>100</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Date</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="Ta_list_2">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>470</y>
     <width>201</width>
     <height>211</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <column>
    <property name="text">
     <string>TA NAME</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>TA ID</string>
    </property>
   </column>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>723</x>
     <y>10</y>
     <width>20</width>
     <height>791</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
